tests/fixtures.py

```diff
@@ -144,3 +144,16 @@ def create_rbd_pod(request):
         interface_type=constants.CEPHBLOCKPOOL,
         pvc_name=class_instance.pvc_obj.name
     )
+
+
+@pytest.fixture()
```

Comment:
I think this is duplicate work of those two PRs:
https://github.com/red-hat-storage/ocs-ci/pull/396/files#diff-8a66b2566ee3ea0285647f603db9174bR159
https://github.com/red-hat-storage/ocs-ci/pull/339/files#diff-10993bfc055d543860f899ca9733c55aR20